src/clipper/argparser.py

```diff
@@ -690,6 +690,14 @@ def getArgParser(clipper_paths: ClipperPaths) -> argparse.ArgumentParser:
         default="yt_dlp",
         help="Choose a youtube_dl alternative for downloading videos.",
     )
+
+    parser.add_argument(
+      "--cookies",
+      "-c",
+      dest="cookies",
+      default="",
+      help="Specify the path to a cookies file to be used by youtube_dl.",
```

Comment:
It may be helpful to update the help text regarding when this feature should be used and maybe the caveat about account bans. A link to relevant yt-dlp docs or issues may also be useful.

Reply:
I am hesitant to commit to code speculative remarks about Youtube's anti-bot measures. Whether the account is banned or not depends on a multitude of factors that change over time, and `cookiefile` option in general serves a broader purpose of accessing content that requires authentication (e.g. member-only videos). Plus, as I understand it, this project supports other video platforms as well.

I have updated the help text to clarify the option's use case in regards to youtube_dl. If you feel it's insufficient, please update the help message in this branch.

Reply:
The help text can be generic but still more informative on how to use the feature and potential caveats. I think if we linked to https://github.com/yt-dlp/yt-dlp/wiki/FAQ#how-do-i-pass-cookies-to-yt-dlp or so it would be sufficient for now.

Reply:
Good idea, I will add the link. Keep in mind, however, that they refer to the param as `cookies` while we ask now to pass `cookiefile`.